sdk/cosmosdb/cosmos/test/public/functional/queryIterator.spec.ts

```diff
@@ -230,3 +249,12 @@ describe("Correlated Activity Id", function () {
     await removeAllDatabases();
   });
 });
+function isValidUUID(capturedCorrelatedIdsList: any[]): boolean {
```

Comment:
change function signature to `isValidUUID(uuid: string): boolean`
Iterate the array in test case itself. 
Also, add a unit test case for this function.

Reply:
move this to utils